### PR TITLE
Import real content from the legacy service

### PR DESCRIPTION
**Pull request by peteryates**
This is a test of [the extractor script](https://github.com/DFE-Digital/support-for-ect-data-extractor/).

- Add clean task to Makefile
- Reorganise for easier handling of multiple parts
- Import sample
- Add support for captioned headings

### CODE REVIEW

**Comment by peteryates** [comment]
Images are now imported, we'll need some stying to ensure they're not too big.

image

File: content/ambition-institute.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+title: "Ambition Institute"
+---
+
+## Year 1
+
+Your module materials
+
+### Autumn term
+
+#### [Behaviour](/ambition-institute/year-1-behaviour)
```
Comment (frankieroberto) [inline content/ambition-institute.md]: @peteryates one to fix in the extractor: we can reduce the number of headings levels by combining the h2 and the h3. The "Your module materials" intro para is also not needed.

```suggestion
## Year 1: Autumn term

### [Behaviour](/ambition-institute/year-1-behaviour)
```

File: content/ambition-institute/year-1-behaviour.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+1. [Video and module introduction](/ambition-institute/year-1-behaviour/autumn-week-1-ect-video-and-module-introduction)
+2. [Module overview](/ambition-institute/year-1-behaviour/autumn-week-1-ect-module-overview)
+3. [Reflect](/ambition-institute/year-1-behaviour/autumn-week-1-ect-reflect)
+
+### Mentor summary
+
+Introduces foundational elements of behaviour and supports teachers and mentors to set up effective ways of working.
+
+[View mentor materials](/ambition-institute/year-1-behaviour/autumn-week-1-mentor-materials)
```
Comment (frankieroberto) [inline content/ambition-institute/year-1-behaviour.md]: Another one for the extractor: we can simplify this by not having the "ECT summary" heading (as I think the default assumption is that the whole service is for ECTs), and then moving all the h4s up to h3s.

This also lets us remove "summary" from the headings, which is kinda redundant, and the "Lessons" heading, which also doesn't feel necessary.

```suggestion
Introduces foundational elements of behaviour and supports teachers and mentors to set up effective ways of working.

1. [Video and module introduction](/ambition-institute/year-1-behaviour/autumn-week-1-ect-video-and-module-introduction)
2. [Module overview](/ambition-institute/year-1-behaviour/autumn-week-1-ect-module-overview)
3. [Reflect](/ambition-institute/year-1-behaviour/autumn-week-1-ect-reflect)

### Mentors

Introduces foundational elements of behaviour and supports teachers and mentors to set up effective ways of working.

[View mentor materials](/ambition-institute/year-1-behaviour/autumn-week-1-mentor-materials)
```

File: layouts/default.slim [hunk 3]
```diff
@@ -13,7 +13,43 @@ html.govuk-template lang='en'
...
-            h1 = item[:title] || 'Insert title here'
+            - if item[:heading]
+              - if item[:caption]
+                span.govuk-caption-l = item[:caption]
+              h1.govuk-heading-l = item[:heading]
+
+            - else
+              - if item[:total_lesson_completion_time]
+                span.govuk-caption-l
+                  | #{item[:total_lesson_completion_time]} minutes
```
Comment (frankieroberto) [inline layouts/default.slim]: Having the estimated lesson completed time as a caption seems quite confusing:

Screenshot 2023-10-16 at 15 58 37

I'm not sure we need to show this on every page - could we bump it up and only show it on the index pages (after the "Lessons" heading) maybe?

**Review by frankieroberto — APPROVED** [review]

**Pull request merged**